Incorrect error message on login screen

Rakshithasai123 opened the pull request:
[RCF-373]

[RCF-373]: https://mosip.atlassian.net/browse/RCF-373?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

* **Bug Fixes**
  * Authentication failures from the server now map to a clear "please try again" retry message, improving error clarity and consistency.

* **Localization**
  * Added and integrated the new authentication-server error variant into localized messages (English, Arabic, French, Hindi, Kannada, Tamil) so users see a translated "unable to get authentication token/please try again" message where applicable.

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

Rakshithasai123 committed:
Incorrect error message on login screen

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
Merge branch 'develop' into RCF-373

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
Update app_ar.arb

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
Update app_en.arb

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
Update app_fr.arb

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
Update app_hi.arb

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
Update app_ta.arb

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
Update app_kn.arb

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

MadhuMosip commented on lib/ui/login_page.dart:
```diff
@@ -353,6 +353,12 @@ class _LoginPageState extends State<LoginPage> with WidgetsBindingObserver {
+    if (errorMsg == "REG_AUTH_SERVER_DOWN" || errorMsg == "KER-SYN-AUTH-001" || errorMsg == "Invalid Request") {
+      _showInSnackBar(appLocalizations.server_unreachable);
+      return;
+    }
```
@Rakshithasai123 needs to handle this with an  error code, not with an error message.

Rakshithasai123 committed:
fixed Incorrect error message on login screen

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

 Conflicts:
	android/app/src/main/java/io/mosip/registration_client/api_services/AuthenticationApi.java

Rakshithasai123 committed:
fixed Incorrect error message on login screen

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
fixed Incorrect error message on login screen

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
fixed Incorrect error message on login screen

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
Refactor login_page.dart for improved readability

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
Refactor login_page.dart for improved readability

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
Merge branch 'develop' of https://github.com/Rakshithasai123/android-registration-client into RCF-373

# Conflicts:
#	assets/l10n/app_ar.arb
#	assets/l10n/app_en.arb
#	assets/l10n/app_fr.arb
#	assets/l10n/app_hi.arb
#	assets/l10n/app_kn.arb
#	assets/l10n/app_ta.arb

Rakshithasai123 committed:
changed the environment

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
Update serverBaseURL in build.gradle

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
Remove comment for ICU-safe error code handling

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
Update error handling for authentication errors

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
resolved conflicts

Rakshithasai123 committed:
Fix missing newline at end of file

Add newline at the end of login_page.dart

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
Remove duplicate entry for 'server_unreachable'

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
Update app_ar.arb

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
Remove unreachable server message from localization

Removed message for unreachable authentication server.

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
Remove duplicate translation entry

Removed duplicate translation for 'server_unreachable'.

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
Remove duplicate 'server_unreachable' entry

Removed duplicate entry for 'server_unreachable' in Hindi localization.

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
Remove duplicate 'server_unreachable' entry

Removed duplicate entry for 'server_unreachable' message.

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
Remove unreachable server error message from localization

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>

Rakshithasai123 committed:
resolved conflicts

Signed-off-by: Rakshithasai123 <rakshithasai2002@gmail.com>
